src/components/MemoryCard.vue

```diff
@@ -38,22 +63,13 @@ export default {
         }
       }
     },
+    openModal(this: { $root: any; $emit: any; cardContent: CardContent }) {
+      eventBus.emit("open-modal", this.cardContent);
```

Comment:
Why did you not use normal emits and handle them in the parent component?

Reply:
Because it would have become to complex. The normal emit only emits to the parent component and I would have to propagate it up multiple levels which would lead to more confusion imo.  So to make the code easier to understand I used an eventBus to directly emit to the target component.

Reply:
I maintain that it will only remain more understandable as long as only one modal is opened via this species and no other modal. Besides, I don't even think it's complex. Only the parent's child says that the content modal should be opened, no other component does this

Reply:
I now changed it to normal emits